twoliter/src/test/data/Twoliter-1.toml

```diff
@@ -3,10 +3,5 @@ release-version = "1.0.0"
 
 [sdk]
 registry = "a.com/b"
-name = "my-bottlerocket-sdk"
-version = "v1.2.3"
-
-[toolchain]
-registry = "c.co/d"
-name = "toolchainz"
-version = "v3.4.5"
+repo = "my-bottlerocket-sdk"
+tag = "v1.2.3"
```

Comment:
@webern we're taking a breaking change for this, so if this isn't the format you actually want, let me know and I can  change it.

Reply:
> we're taking a breaking change for this, so if this isn't the format you actually want, let me know and I can change it.

The only question I have is whether we would prefer `sdk = "a.com/b/my-bottlerocket-sdk:v1.2.3"` since we no longer need to do any processing on this. I can go either way, doesn't seem important.

Reply:
I think a single sdk string would create a clearer interface.

Reply:
> The only question I have is whether we would prefer sdk = "a.com/b/my-bottlerocket-sdk:v1.2.3" since we no longer need to do any processing on this.

@webern would you want this to become a top-level key rather than a nested map? E.g.
```
❯ cat Twoliter.toml
schema-version = 1
release-version = "1.19.2"
sdk = "a.com/b/my-bottlerocket-sdk:v1.2.3"
```